src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigMapResource.java

```diff
@@ -62,6 +67,12 @@ public ResponseEntity<Void> updateConfiguration(@RequestBody Configuration confi
         return ResponseEntity.ok().build();
     }
 
+    private void assertUpdateConfigurationAvailable() {
+        if (!applicationProperties.isUpdateConfigAvailable()) {
+            throw new BusinessException("Configuration update is not available");
```

Comment:
better to throw AccessDeniedException (403)

Reply:
AccessDeniedException is unheadeled, is it ok?

Reply:
done